### PR TITLE
CIRC-6179 - IRONdb 0.19.16 Release Notes

### PR DESCRIPTION
**Pull request by pamaddox**

### CODE REVIEW

File: content/irondb/release-notes.md [hunk 1]
```diff
@@ -5,6 +5,28 @@ weight: 40
 
 # Release Notes
 
+## Changes in 0.19.16
+
+2021-01-29
+
+ * Added `graphite:aliassub` in CAQL to emulate graphite's aliasSub function.
```
Comment (esproul) [inline content/irondb/release-notes.md:12]:
```suggestion
 * Added `graphite:aliassub` in CAQL to emulate Graphite's aliasSub function.
```

File: content/irondb/release-notes.md [hunk 2]
```diff
@@ -5,6 +5,28 @@ weight: 40
...
+ * Added `stats:ratio(of=1)` in CAQL to allow calculating each input stream over the
+   sum of streams.
+ * Added optional verbose rollup/delete debug logging
+ * Fix race in raw data rollups where we could attempt to roll data up
+   before opening the db.
+ * Support vectorized operations across most CAQL functions (stats and ops)
+ * Add several graphite-style functions to assist those migrating from graphite
```
Comment (esproul) [inline content/irondb/release-notes.md:19]:
```suggestion
 * Add several Graphite-style functions to assist those migrating from Graphite
```

**Review by esproul — CHANGES_REQUESTED** [review]
Capitalize "Graphite" since we are referring to a product name.

**Review by esproul — APPROVED** [review]
LGTM

File: content/irondb/release-notes.md [hunk 3]
```diff
@@ -5,6 +5,30 @@ weight: 40
...
+
+2021-01-29
+
+ * Added `graphite:aliassub` in CAQL to emulate Graphite's aliasSub function.
+ * Added `stats:ratio(of=1)` in CAQL to allow calculating each input stream over the
+   sum of streams.
+ * Added optional verbose rollup/delete debug logging
+ * Fix race in raw data rollups where we could attempt to roll data up
+   before opening the db.
+ * Fix bug in raw data rollups where if certain metrics were removed from the
```
Comment (esproul) [inline content/irondb/release-notes.md:18]: "certain metrics" makes it sound like they needed to match some kind of criteria to cause the bug; I thought the issue was _any_ deleted metrics.

Comment (pamaddox) [reply content/irondb/release-notes.md:18]: Depends on when and where the metric is deleted. If you delete a metric and we've already passed it in the rollup, there will be no impact. Regardless - I'll change to just "metrics are deleted"

Comment (pamaddox) [reply content/irondb/release-notes.md:18]: Also - if you remove a metric from the system and it's not part of a rollup that needs to be done, it won't have any impact. It's really "If we delete a surrogate ID from the system AND it's in a shard that needs to be rolled up AND we haven't passed that surrogate yet in the process", but I don't think that will be comprehensible to users.

Comment (pamaddox) [reply content/irondb/release-notes.md:18]: I'll say "full delete", which is the process by which we remove surrogates without removing the data itself, which is what triggers this.

Comment (pamaddox) [reply content/irondb/release-notes.md]: Fixed

**Review by esproul — APPROVED** [review]
LGTM

**Pull request merged**